source/agora/protocol/GossipProtocol.d

```diff
@@ -0,0 +1,97 @@
+/*******************************************************************************
+
+    Contains the code used for peer-to-peer communication.
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.protocol.GossipProtocol;
```

Comment:
I'm not convinced we need a `protocol` package at this stage. I think we should apply [the rule of three](https://blog.codinghorror.com/rule-of-three/) here and introduce the `protocol` package when we have 3 users for it. What do you think ?

Reply:
@Geod24
Check Please ~